Traceback (most recent call last):
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/train.py", line 95, in <module>
    exp.run()
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/diffusion/experiment.py", line 154, in run
    super(DiffusionExperiment, self).run(epochs=self.args.epochs)
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/diffusion/base.py", line 181, in run
    train_dict = self.train_fn(epoch)
                 ^^^^^^^^^^^^^^^^^^^^
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/experiment.py", line 32, in train_fn
    loss = elbo_bpd(self.model, pyg_data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/diffusion/loss.py", line 29, in elbo_bpd
    return loglik_bpd(model, x)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/diffusion/loss.py", line 12, in loglik_bpd
    return -model.log_prob(x).sum() / (math.log(2) * x.num_entries)
            ^^^^^^^^^^^^^^^^^
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/diffusion/diffusion_base.py", line 185, in log_prob
    return self._train_loss(batched_graph)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yuliaterzieva/Desktop/Thesis - Diffusion/Code/MScThesis/EDGE/diffusion/diffusion_binomial_active.py", line 264, in _train_loss
    raise ValueError()
ValueError
